causalml/inference/meta/rlearner.py

```diff
@@ -158,10 +159,10 @@ def fit_predict(self, X, p, treatment, y, return_ci=False,
             return te
         else:
             start = pd.datetime.today()
-            self.t_groups_global = self.t_groups
-            self._classes_global = self._classes
-            self.model_mu_global = deepcopy(self.model_mu)
-            self.models_tau_global = deepcopy(self.models_tau)
+            t_groups_global = self.t_groups
+            _classes_global = self._classes
+            model_mu_global = deepcopy(self.model_mu)
+            models_tau_global = deepcopy(self.models_tau)
```

Comment:
Let's use `tqdm` in the `for` loop below as well.

Reply:
Yea, forgot that part. Just changed. Thanks!